src/cl-giskard.lisp

```diff
@@ -50,58 +50,67 @@
 (defparameter *giskard-action-goal-type* "giskard_msgs/WholeBodyCommand")
 
 (defparameter *tf-listener* nil)
```

Comment:
I recommend to avoid this type of global variables because they introduce side-effects. 

Also, they make it hard to use the functionality provided by ```cl-giskard``` twice within the same application. I know this does not happen often, but we had such use-cases.